apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -129,7 +129,7 @@ const TutorialExplorer = React.createClass({
    */
   componentDidUpdate() {
     if (this.shouldScrollToTop) {
-      $('html, body').animate({scrollTop: $("#tutorials").offset().top});
+      $('html, body').animate({scrollTop: $(this.allTutorials).offset().top});
       this.shouldScrollToTop = false;
```

Comment:
Realize this was done in a separate PR, but might be good to have a comment somewhere explaining how shouldScrollToTop works, given that the approach is a little non-standard. It's also worth noting that the contract of your scrollToTop function is just that it will result in a scroll *on the next render*. If that next render doesn't happen (and we dont hit componentDidUpdate) we'll never scroll.

The actual change in this PR lgtm :)